core/src/main/java/tc/oc/pgm/structure/Dynamic.java

```diff
@@ -0,0 +1,69 @@
+package tc.oc.pgm.structure;
+
+import org.bukkit.util.Vector;
+import tc.oc.pgm.api.feature.Feature;
+import tc.oc.pgm.api.match.Match;
+import tc.oc.pgm.util.BudgetWorldEdit;
+
+public class Dynamic implements Feature<DynamicDefinition> {
+
+  private final Match match;
+  final StructureDefinition structureDefinition;
+  final Vector offset;
+  private final DynamicDefinition dynamicDefinition;
+
+  // Since the passive filter can skip placing the structure,
+  // we need to keep track of whether its placed or not if we
+  // want to avoid unnecessary clears.
+  boolean placed;
```

Comment:
```suggestion
  private boolean placed;
```
same thing here, if others want to check this, do so via getter